README.md

```diff
@@ -170,3 +126,82 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+# Deployment to Fly.io
+
+## Install FlyCtl
+
+To work with the Fly platform, you first need to install Flyctl, a command line interface that allows you to do everything from creating an account to deploy to Fly.
+  - Linux:
+    ```  
+    curl  -L https://fly.io/install.sh | sh
+    ```
+  - OSX:
+    ```
+    brew install flyctl
+    ```
+  - Windows:
+    ```
+    iwr https://fly.io/install.ps1 -useb
+    ```  
+    Here is the [Installation Guilde](https://fly.io/docs/hands-on/install-flyctl/)
+
+
+Next authenticate with your fly.io account:
+  ```
+  fly auth login
+  ```
+
+- To make sure everything is working well:    
+  ```
+  fly apps list
+  ```
+The output of the above command will be empty table since you have no apps launched yet
+
+
+## Configure the Project
+
+### Environment Variables
+
+We shouldn't store secrets in source code, so utilizing environmental variables is needed.
+- Run the following from the root of your project
+  ```
+  python3 set_env.py
+  ```
+
+### Deploy App
+
+In this step the app is going to be launched to fly.io.
+- Create and configure the app  
+  ```
+  python fly_manager.py launch <app-name>
+  
```

Comment:
This asks a million questions. Isn't it possible to answer them automatically? Or at least document what all the answers should be?

Reply:
I was trying to reply authomatically , i did for some of them using arguments but still it will ask some. i will put the answers

Reply:
The logs say: "Save your credentials in a secure place -- you won't be able to see them again!" for postgres. It has a DIFFERENT password than DATABASE_URL. Do I need the first set of credentials, yes or no? If yes, where should they be stored in the repo, so they are not included in git or the docker? This should be documented

Reply:
The logs say: "Save your credentials in a secure place -- you won't be able to see them again!" for postgres. It has a DIFFERENT password than DATABASE_URL. Do I need the first set of credentials, yes or no? If yes, where should they be stored in the repo, so they are not included in git or the docker? This should be documented

Reply:
Yes, I was storing this on tmp manually before(if you remember it), now this is where i stuck .. i am not getting this json response with script. any help please?

Reply:
I specified those vars from my side , we can store in .env file like DATABASE_URL

Reply:
it's automatic, no questoinaire

Reply:
I haven't verified but last time I ran this command it asked a million questions. Either we answer automatically through script or the answers should be documented